app/controllers/prior_authority/steps/additional_costs_controller.rb

```diff
@@ -0,0 +1,25 @@
+module PriorAuthority
+  module Steps
+    class AdditionalCostsController < BaseController
+      def edit
+        @form_object = AdditionalCosts::OverviewForm.build(
+          current_application
+        )
+        # We never want 'yes' to be pre-selected as it makes for confusing UX.
+        # 'No' being pre-selected is fine otherwise users have to re-enter the same information
+        # every time they come back to this screen.
```

Comment:
if there is a test that locks this behaviour down I would not bother with the comment but no bother either way